addons/calendar/static/src/views/fields/many2many_attendee_expandable.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="calendar.Many2ManyAttendeeExpandable" t-inherit="web.Many2ManyTagsAvatarField" t-inherit-mode="primary" owl="1">
+        <xpath expr="div" position="attributes">
```

Comment:
I would add an `id` to the div in web.Many2ManyTagsAvatarField and target it here (no need to use a xpath). 
```xml
<div id="my_id" position="attributes">
```

That way, you are more selective and there will be no ambiguity if the web.Many2ManyTagsAvatarField component evolves in the future.